cfme/common/vm.py

```diff
@@ -442,6 +442,30 @@ def unset_ownership(self):
         view = self.create_view(navigator.get_class(self, 'Details').VIEW)
         view.flash.assert_success_message(msg)
 
+    def rename_vm(self, new_vm_name, click_cancel=False, click_reset=False):
+        """Rename the VM
+
+        Args:
+            new_vm_name: object for renaming vm
+            click_cancel (bool): Whether to cancel form submission
+            click_reset (bool): Whether to reset form after filling
+        """
+        view = navigate_to(self, 'Rename')
+        view.form.fill({'vm_name': new_vm_name})
+        if click_reset:
+            view.form.reset_button.click()
+            view.flash.assert_no_error()
+            view.form.cancel_button.click()
+
+        elif click_cancel:
+            view.form.cancel_button.click()
+            view.flash.assert_no_error()
+
+        else:
+            # save the form
+            view.form.save_button.click()
```

Comment:
Required:
After rename VM. we need to return `VM object` with the `new name` (instantiate)